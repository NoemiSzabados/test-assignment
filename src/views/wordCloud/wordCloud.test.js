
import renderer from 'react-test-renderer';
import WordCloudContainer from "./index";
import ReactDOM from "react-dom";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe('Word Cloud', () => {
    it('snapshot test', () => {
        const component = renderer.create(
            <WordCloudContainer />,
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();

    });

    it('renders without crush', () => {
        const div = document.createElement("div");
        ReactDOM.render(<WordCloudContainer />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

});
