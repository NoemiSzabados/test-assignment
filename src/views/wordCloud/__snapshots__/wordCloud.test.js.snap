// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Word Cloud snapshot test 1`] = `
<div
  className="wordCloudWrapper"
>
  <div
    className="wordCloudContainer"
  />
</div>
`;
